### ISSUE TITLE
XBOX 360 L-Analog stick + Atari 2600

### ISSUE THREAD

**Issue by gardobeson** [RetroPie/RetroPie-Setup#229]
Greetings Chaps,

I am trying to get my retroarch.cfg for the Atari 2600 to take the customizations that I've made to override the /All/retroarch.cfg file.  Here's the XBOX 360 part of the /All/retroarch.cfg:

input_device = "Microsoft X-Box 360 pad"
input_driver = "linuxraw"
input_b_btn = "1"
input_y_btn = "3"
input_select_btn = "6"
input_start_btn = "7"
input_up_axis = "-7"
input_down_axis = "+7"
input_left_axis = "-6"
input_right_axis = "+6"
input_a_btn = "0"
input_x_btn = "2"
input_l_btn = "4"
input_r_btn = "5"
input_l2_axis = "+2"
input_r2_axis = "+5"
input_l3_btn = "9"
input_r3_btn = "10"
input_l_x_plus_axis = "+0"
input_l_x_minus_axis = "-0"
input_l_y_plus_axis = "+1"
input_l_y_minus_axis = "-1"
input_r_x_plus_axis = "+3"
input_r_x_minus_axis = "-3"
input_r_y_plus_axis = "+4"
input_r_y_minus_axis = "-4"

Here's what I've done to the /atari2600/retroarch.cfg

input_device = "Microsoft X-Box 360 pad"
input_driver = "linuxraw"
input_b_btn = "1"
input_y_btn = "3"
input_select_btn = "6"
input_start_btn = "7"
input_a_btn = "0"
input_x_btn = "2"
input_l_btn = "4"
input_r_btn = "5"
input_l2_axis = "+2"
input_r2_axis = "+5"
input_l3_btn = "9"
input_r3_btn = "10"
input_l_x_plus_axis = "+6"
input_l_x_minus_axis = "-6"
input_l_y_plus_axis = "-7"
input_l_y_minus_axis = "+7"
input_r_x_plus_axis = "+3"
input_r_x_minus_axis = "-3"
input_r_y_plus_axis = "+4"
input_r_y_minus_axis = "-4"

Basically, I took out the U, D, L, R and their values and gave them to the input_l_x_plus_axis, etc.  

When I open the 2600 emulator, it uses the /All/retroarch.cfg info and ignores the changes.  Any help would be greatly appreciated.  Let me know if you need more info.

-Gardo
